src/formio/components/FileField.js

```diff
@@ -189,6 +189,12 @@ class FileField extends Formio.Components.components.file {
     super.upload(files);
   }
 
+  browseFiles(attrs = {}) {
+    // provide class name for hiding element, override style to avoid triggering CSP
+    attrs = {class: 'fileUploadInput', style: ''};
```

Comment:
please prefix the classname (use the `prefix` utility function) and use dashes in the classname rather than camelCase to be consistent with the rest of the classnames used by OF.